### DISCUSSION TITLE
Welcome to tsv-utils Discussions!

### DISCUSSION THREAD

**Discussion by jondegenhardt** [eBay/tsv-utils#324] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jeffreycollett** [comment]
Is there a way to pivot a table with this tool?

from:
A     B      C
1      2      3
4     5      6

to:
A     1     4
B     2     5
C     3    6

Thanks!

**Reply by jondegenhardt** [reply]
Hi @jeffreycollett,

No, tsv-utils doesn't have row-column pivoting. You could try GNU datamash. Its `transpose` operation does exactly this.

--Jon

**Reply by jeffreycollett** [reply]
Jon, datamash works great, thank you for the suggestion. Also, thank you for the great TSV tools!
